spec/google_export_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require_relative './../lib/lsp-data'
+require 'spec_helper'
+require 'byebug'
```

Comment:
Just a thought for the future, you could require 'byebug' inside the spec_helper file, so it's ready for you when you need it without having to require it in the individual test files.